<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FmtEditContentFilter</class>
 <widget class="QDialog" name="FmtEditContentFilter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Параметры фильтрации/сортировки</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res/fmt.qrc">
    <normaloff>:/img/Filter2HS.png</normaloff>:/img/Filter2HS.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Сортировка</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,1,0">
       <item>
        <widget class="QListView" name="fldListView"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="removeFromOrder">
           <property name="text">
            <string>&lt;</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="addAsc">
           <property name="text">
            <string>&gt; (Asc)</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="addDesc">
           <property name="text">
            <string>&gt; (Desc)</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListView" name="orderListView">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="orderBtnUp">
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>▲</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="orderBtnDown">
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>▼</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>30</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Фильтр</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout" columnstretch="0,1,0,1,0,0,1">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="1" column="4">
           <widget class="QToolButton" name="condToolButton2">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="fldComboBox">
            <property name="editable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QToolButton" name="condToolButton1">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="0" column="5">
           <widget class="QPushButton" name="pushAnd">
            <property name="text">
             <string>And</string>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLineEdit" name="condLineEdit1"/>
          </item>
          <item row="0" column="2">
           <widget class="QComboBox" name="condComboBox">
            <property name="editable">
             <bool>true</bool>
            </property>
            <item>
             <property name="text">
              <string>=</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>&lt;&gt;</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>&gt;</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>&gt;=</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>&lt;</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>&lt;=</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>LIKE</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>BETWEEN</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>IS NOT NULL</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>IS NULL</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>IN</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>NOT IN</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLineEdit" name="condLineEdit2"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Колонки:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="5">
           <widget class="QPushButton" name="pushOr">
            <property name="text">
             <string>Or</string>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="6">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="6">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res/fmt.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FmtEditContentFilter</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FmtEditContentFilter</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
